doc/api/path.md

```diff
@@ -155,11 +223,15 @@ path.format({
 
 ## path.isAbsolute(path)
 
-Determines whether `path` is an absolute path. An absolute path will always
-resolve to the same location, regardless of the working directory. `path` must
-be a string.
+* `path` {String} A file path
 
-Examples on \*nix:
+The `path.isAbsolute()` method determines if `path` is an absolute path. An
+absolute path will always resolve to the same location, regardless of the
```

Comment:
I know this was copied from the original wording, but this could be a bit misleading when an absolute path containing symlinks is used. It may be better to just drop that sentence entirely?